<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Frank Osterfeld</author>
 <class>Akregator::SettingsArchive</class>
 <widget class="QWidget" name="Akregator::SettingsArchive">
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Default Archive Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="3" column="0">
       <widget class="QRadioButton" name="rb_DisableArchiving">
        <property name="text">
         <string>Disable archiving</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_KeepAllArticles">
        <property name="text">
         <string>Keep all articles</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rb_LimitArticleNumber">
          <property name="text">
           <string>Limit feed archive size to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_MaxArticleNumber">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="specialValueText">
           <string>1 article</string>
          </property>
          <property name="suffix">
           <string> articles</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rb_LimitArticleAge">
          <property name="text">
           <string>Delete articles older than: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_MaxArticleAge">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="specialValueText">
           <string>1 day</string>
          </property>
          <property name="suffix">
           <string> days</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_DoNotExpireImportantArticles">
     <property name="text">
      <string>Do not expire important articles</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
